verilog/CST/package.cc

```diff
@@ -39,4 +39,9 @@ const verible::TokenInfo& GetPackageNameToken(const verible::Symbol& s) {
   return name_node.get();
 }
 
+const verible::SyntaxTreeLeaf* GetPackageNameToken2(const verible::Symbol& s)
```

Comment:
Suggest naming this function `GetPackageNameLeaf` and have it return `const verible::SyntaxTreeLeaf&`, and have `GetPackageNameToken` call this one (to avoid replicating logic).

Reply:
changed the function as requested, also added more testcases